cnf-certification-test/networking/suite.go

```diff
@@ -121,8 +121,30 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
 		testPartnerSpecificTCPPorts(&env)
 	})
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestDpdkCPUPinningExecProbe)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
+		testExecProbDenyAtCPUPinning(&env)
+	})
 })
 
+func testExecProbDenyAtCPUPinning(env *provider.TestEnvironment) {
```

Comment:
Change:
func testExecProbDenyAtCPUPinning(env *provider.TestEnvironment) {
	ginkgo.By("Check if DPDK pod with cpu pinning is running")

	dpdkPods := env.GetCPUPinningPodsWithDpdk()
	testhelper.SkipIfEmptyAny(ginkgo.Skip, dpdkPods)
to:
func testExecProbDenyAtCPUPinning(env *provider.TestEnvironment, dpdkPods []*provider.Pod) {